woocommerce/woocommerce-gutenberg-products-block
# [Store Customization] Update the Centered Header Menu with Search Pattern

## What

Fixes a few issues with the `Centered Header Menu with Search`:
- Fix margins in the separator and first row.
- Increase navigation items spacing.
- Fix hidden search, cart, and account in mobile.
- Update site logo width

Fixes https://github.com/woocommerce/woocommerce-blocks/issues/11253
Fixes https://github.com/woocommerce/woocommerce-blocks/issues/11389

## Why

To match the design requirements in https://github.com/woocommerce/woocommerce-blocks/issues/11253

## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Create a new page or post.
2. Insert the `Centered Header Menu with Search` pattern.
3. Check it looks like the screenshot below.
4. Check all the elements are visible on mobile.


## Screenshots or screencast



| Desktop |
|--------|
| <img width="1351" alt="Screenshot 2023-10-23 at 14 04 18" src="https://github.com/woocommerce/woocommerce-blocks/assets/186112/69a1cc45-b557-4243-b071-ad9e75f18590"> |

| Mobile |
|--------|
| <img width="424" alt="Screenshot 2023-10-23 at 14 09 48" src="https://github.com/woocommerce/woocommerce-blocks/assets/186112/f6cc07ad-e8f7-4e1f-bfbc-774f5103e408"> |

## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Improve the `Centered Header Menu with Search` pattern.
